app/updatemanager.go

```diff
@@ -0,0 +1,163 @@
+// Copyright 2021 Northern.tech AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
+package app
+
+import (
+	"context"
+	"encoding/json"
+	"strconv"
+	"sync"
+
+	"github.com/mendersoftware/mender/dbus"
+
+	"github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
+)
+
+const (
+	UpdateManagerDBusPath          = "/io/mender/UpdateManager"
+	UpdateManagerDBusObjectName    = "io.mender.UpdateManager"
+	UpdateManagerDBusInterfaceName = "io.mender.Update1"
+	UpdateManagerDBusInterface     = `
+        <node>
+            <interface name="io.mender.Update1">
+                <method name="SetUpdateControlMap">
+                    <arg type="s" name="update_control_map" direction="in"/>
+                    <arg type="i" name="refresh_timeout" direction="out"/>
+                 </method>
+	    </interface>
```

Comment:
Nitpick: Indentation.

Reply:
This was resolved, but not fixed. Not sure if you meant that you had it locally or not, but just in case you forgot.

Reply:
I stared at this thing for the longest time, and found one indent which was 3 spaces, instead of 4. I guess that was not what you meant then? 
I'm re-indenting the whole thing with four spaces to indent it away from the variable declarations, if this is not what you mean still, let's just fix it when you're back :)